test/tap/LoggingEvent-test.js

```diff
@@ -97,5 +97,176 @@ test('LoggingEvent', (batch) => {
+    const fileName = '/log4js-node/test/tap/layouts-test.js';
+    const lineNumber = 1;
+    const columnNumber = 14;
+    const functionName = 'foo';
+    const functionAlias = 'bar';
+    const className = '';
+    const callerName = 'foo [as bar]';
+    const location = {
+      functionName,
+      fileName,
+      lineNumber,
+      columnNumber,
+      callStack,
+      className,
+      functionAlias,
+      callerName,
+    };
+    const event = new LoggingEvent(
+      'cheese',
+      levels.DEBUG,
+      ['log message'],
+      { user: 'bob' },
+      location
+    );
+    t.equal(event.functionName, functionName);
+    t.equal(event.fileName, fileName);
+    t.equal(event.lineNumber, lineNumber);
+    t.equal(event.columnNumber, columnNumber);
+    t.equal(event.callStack, callStack);
+    t.equal(event.className, '');
+    t.equal(event.functionAlias, functionAlias);
+    t.equal(event.callerName, callerName);
+    t.end();
+  });
+
+  batch.test('Should contain class and method names', (t) => {
+    // console.log([Error('123').stack.split('\n').slice(1).join('\n')])
+    const callStack =
+      '    at Foo.bar (repl:1:14)\n    at ContextifyScript.Script.runInThisContext (vm.js:50:33)\n    at REPLServer.defaultEval (repl.js:240:29)\n    at bound (domain.js:301:14)\n    at REPLServer.runBound [as eval] (domain.js:314:12)\n    at REPLServer.onLine (repl.js:468:10)\n    at emitOne (events.js:121:20)\n    at REPLServer.emit (events.js:211:7)\n    at REPLServer.Interface._onLine (readline.js:280:10)\n    at REPLServer.Interface._line (readline.js:629:8)'; // eslint-disable-line max-len
+    const fileName = '/log4js-node/test/tap/layouts-test.js';
+    const lineNumber = 1;
+    const columnNumber = 14;
+    const className = 'Foo';
+    const functionName = 'bar';
+    const functionAlias = '';
+    const callerName = 'Foo.bar';
+    const location = {
+      functionName,
+      fileName,
+      lineNumber,
+      columnNumber,
+      callStack,
+      className,
+      functionAlias,
+      callerName,
+    };
+    const event = new LoggingEvent(
+      'cheese',
+      levels.DEBUG,
+      ['log message'],
+      { user: 'bob' },
+      location
+    );
+    t.equal(event.functionName, functionName);
+    t.equal(event.fileName, fileName);
+    t.equal(event.lineNumber, lineNumber);
+    t.equal(event.columnNumber, columnNumber);
+    t.equal(event.callStack, callStack);
+    t.equal(event.className, className);
+    t.equal(event.callerName, callerName);
+    t.equal(event.functionAlias, '');
+    t.end();
+  });
+
```

Comment:
I think there is no need for these test cases as it's a straightforward variable assignment.
The `Should contain class, method and alias names` test case would have covered them.

```suggestion
```

Reply:
Removed all three in commit [5f988bd](https://github.com/log4js-node/log4js-node/pull/1316/commits/5f988bd097b4b5218fd564a2a1cd0466c5501f0f)